src/index.html

```diff
@@ -11,7 +11,219 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-div-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+
+      <fieldset class="personalinfo">
+        <legend>
+          Personal information
+        </legend>
+
+        <label>
+          Surname:  
+          <input 
+            type="text"
+            class="surname" 
+            name="surname" 
+            required
+          >
+        </label>
+
+        <div class="one">  
+          <label>
+            Name:  
+            <input 
+              type="text"
+              class="name"
+              name="name" 
+              required
+            >      
+          </label>
+        </div>
```

Comment:
Classes should have more meaningful names. And do you really need additional `div`s everywhere? 

```suggestion
        <label class="form-field">
          Name:  
          <input 
            type="text"
            class="name"
            name="name" 
            required
          >      
        </label>
```

Reply:
labels is inline elements. They don't make blocks